explorer/client/src/components/ownedobjects/OwnedObjects.tsx

```diff
@@ -128,7 +135,38 @@ function OwnedObjectView({ results }: { results: resultType }) {
     );
 }
 
-function OwnedObject({ objects }: { objects: string[] }) {
+function GetObjectsStatic({ id }: { id: string }) {
+    const objects = findOwnedObjectsfromID(id);
+
+    if (objects !== undefined) {
+        return (
+            <OwnedObjectSection
+                objects={objects.map(({ objectId }) => objectId)}
+            />
+        );
+    }
+
+    return <div />;
+}
+
+function GetObjectsAPI({ id }: { id: string }) {
+    const [objects, setObjects] = useState([{ objectId: 'Please Wait' }]);
```

Comment:
import type { ObjectRef } from 'sui.js';

```suggestion
    const [objects, setObjects] = useState<ObjectRef[]>([]);
```

This will help you get rid of the error